playwright/accessibility/foundations__messaging__overview.spec.mjs

```diff
@@ -2,7 +2,7 @@
 import { test } from '@playwright/test';
 import expectAccessiblePage from './expectAccessiblePage.mjs';
 
-test('Data viz palette check', async ({ page }) => {
-  await page.goto('/foundations/data_visualization/palette');
+test('Messaging overview accessibility check', async ({ page }) => {
```

Comment:
git's being weird here. This wasn't converting a test to a different path — probably from a duplicate test I deleted